ontology/yaml/resources/LIGHTING/entity_types/GENERALTYPES.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+LT:
+  description: "Luminaire, lighting fixture."
+  implements:
+  - EQUIPMENT
+  uses:
+  - dimmer_percentage_command # in % indicating brightness
+
+LPS:
+  description: "Lighting photocell sensor."
+  implements:
+  - MTS	
+
+LMS:
+  description: "Lighting motion sensor."
+  implements:
+  - MTS	
+
+LTGW:
+  description: "Lighting gateway."
+  implements:
+  - EQUIPMENT			
+  
+LAC:
+  description: "lighting area controller."
+  implements:
+  - EQUIPMENT	
+
+LCM:
```

Comment:
Should this just be lighting area controller? Not enough information in the description to understand why we need both.

Reply:
See this [diagram](https://docs.google.com/presentation/d/1u-eQqeKscl5VmFzQqHUHISr1nWVuUg-JrMm_jiOkwUw/edit#slide=id.g89fd74b19d_0_5) for reference. It could potentially become a more generic "lighting controller" if we drop the spatial/topology attribute of being on a different level of the communication hierarchy.